Jenkinsfile

```diff
@@ -120,8 +121,35 @@ pipeline {
         container('base') {
           nxWithGitHubStatus(context: 'release', message: 'Release') {
             script {
-              // tag the version - nuxeo-helm-chart doesn't follow the v1.2.3 convention
-              nxGit.commitTagPush(tag: env.VERSION)
+              nxGit.commitTagPush()
+            }
+          }
+        }
+      }
+    }
+    stage('Release Jira version') {
+      when {
+        expression { !nxUtils.isPullRequest() }
+      }
+      steps {
+        container('base') {
+          script {
+            def jiraVersionName = "helm-chart-${VERSION}"
+            // create a new released version in Jira
+            def jiraVersion = [
+                project: 'NXP',
+                name: jiraVersionName,
+                releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
+                released: true,
+            ]
+            nxJira.newVersion(version: jiraVersion)
+            // find Jira tickets included in this release and update them
+            def jiraTickets = nxJira.jqlSearch(jql: "project = NXP and fixVersion = '${JIRA_NUXEO_CHART_MOVING_VERSION}'")
+            def previousVersion = sh(returnStdout: true, script: "perl -pe 's/\\b(\\d+)(?=\\D*\$)/\$1-1/e' <<< ${VERSION}").trim()
```

Comment:
Do we want to take care of a new minor or major version, aka 3.2.0 or 4.0.0?

Reply:
If I remember correctly, the perl script will produce a wrong version, like `4.0.-1`, but it is caught by the git change log method that fallback on a full retrieval of history.
In the end, if nothing fail, this code is more or less optimization for git commands.
🤞